.github/workflows/packtracker.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         if: github.event_name == 'pull_request'
         run: |
           sbt seqexec_web_client/fullOptJS::webpack
-          cd modules/seqexec/web/client/target/scala-2.12/scalajs-bundler/main/
+          cd modules/seqexec/web/client/target/scala-2.13/scalajs-bundler/main/
```

Comment:
Can't you use a variable for the version number, here and in all instances?

Reply:
We would have to somehow extract Scala's version number from sbt. Either that, or execute packtracker as an sbt task.

Reply:
I’m not sure it is worth the hassle